src/platform/forms-system/src/js/components/ExpandingGroup.jsx

```diff
@@ -18,17 +19,29 @@ export default function ExpandingGroup({
   showPlus,
   additionalClass,
   expandedContentId,
+  expandedContentFocus,
 }) {
   const classNames = classnames(
     'form-expanding-group',
-    { 'form-expanding-group-open': open },
+    { 'form-expanding-group-open': open && !expandedContentFocus },
     { 'form-expanding-group-plus': showPlus },
   );
 
+  useEffect(
+    () => {
+      if (open && expandedContentFocus) {
+        scrollAndFocus(document.querySelector('.form-expanding-group-open'));
```

Comment:
related to this, because there is a similar problem when you add a new array item, I opened a ticket for focusing the first element in a array pattern regardless if web component or not - if we want to try to also tackle this at the same time, we could add it to this - https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team/62008